### DISCUSSION TITLE
Trees everywhere

### DISCUSSION THREAD

**Discussion by sinistre1973** [kubilus1/autoortho#208] [Q&A]
I think this is a XP12 problem, but anybody knows how to get rid of them? Running XP12.05 64-Bit Vulkan

**Comment by sinistre1973** [comment]
OK, X-Europe from Simheaven seems to do the trick. I only installed the vegetation library + simheaven_X-world_xxxxx-7-forests. Looks better now. Will do this for USA too

**Comment by dpdurst** [comment]
The Trees everywhere is an XP12 issue not an AO issue. As you noticed Sim Heaven will resolve the tree issue
